### ISSUE TITLE
fix: cannot deselect an option if there is initially 1 selected in AutoComplete

### ISSUE THREAD

**Issue by lasbrojen** [microsoft/fluentui-blazor#2907]
Hi 🙂

# 🐛 Bug Report

In AutoComplete, you cannot deselect an option if there is initially 1 selected.

I've included code and a GIF below for reference.

 If you add this back and initially have 2 selected it works:
![Image](https://github.com/user-attachments/assets/62675d3c-eabb-4dae-a47e-af96049ed892)

https://github.com/user-attachments/assets/b55c6bec-e5b9-4865-a252-60e69c9b89b4

Thank you for this nugget ✌️

## 💻 Repro or Code Sample
```
<FluentAutocomplete 
    TOption="Car"
    Label="Select Car"
    KeepOpen="true"
    Multiple="true"
    MaxAutoHeight="300px"
    AutoComplete="off"
    Placeholder="Select cars"
    Width="250px"
    OnOptionsSearch="OnSearch"
    MaximumOptionsSearch="int.MaxValue"
    MaximumSelectedOptions="int.MaxValue"
    Virtualize="true"
    OptionText="@(item => $"{item.Name}")"
    @bind-SelectedOptions="@SelectedCars" />
@code
{    
    private IEnumerable<Car> SelectedCars { get; set; }

    private void OnSearch(OptionsSearchEventArgs<Car> e)
    {
        e.Items = _softCachedCars.Where(car =>
            car.Name.Contains(e.Text, StringComparison.OrdinalIgnoreCase));
    }

    private IReadOnlyCollection<Car> _softCachedCars = new List<Car>
    {
        new Car { Id = 1, Name = "Audi" },
        new Car { Id = 2, Name = "BMW" },
        new Car { Id = 3, Name = "Mercedes" },
        new Car { Id = 4, Name = "Toyota" },
        new Car { Id = 5, Name = "Ford" },
        new Car { Id = 6, Name = "Volkswagen" },
        new Car { Id = 7, Name = "Renault" },
        new Car { Id = 8, Name = "Peugeot" },
        new Car { Id = 9, Name = "Fiat" },
        new Car { Id = 10, Name = "Opel" },
        new Car { Id = 11, Name = "Citroen" },
        new Car { Id = 12, Name = "Hyundai" },
        new Car { Id = 13, Name = "Kia" },
        new Car { Id = 14, Name = "Mazda" },
        new Car { Id = 15, Name = "Skoda" },
        new Car { Id = 16, Name = "Seat" },
        new Car { Id = 17, Name = "Volvo" },
        new Car { Id = 18, Name = "Honda" },
        new Car { Id = 19, Name = "Nissan" },
        new Car { Id = 20, Name = "Suzuki" },
        new Car { Id = 21, Name = "Mitsubishi" },
        new Car { Id = 22, Name = "Subaru" },
        new Car { Id = 23, Name = "Chevrolet" },
        new Car { Id = 24, Name = "Dacia" },
        new Car { Id = 25, Name = "Jeep" },
        new Car { Id = 26, Name = "Land Rover" },
        new Car { Id = 27, Name = "Jaguar" },
        new Car { Id = 28, Name = "Lexus" },
        new Car { Id = 29, Name = "Porsche" },
        new Car { Id = 30, Name = "Mini" },
        new Car { Id = 31, Name = "Alfa Romeo" },
        new Car { Id = 32, Name = "Smart" },
        new Car { Id = 33, Name = "Saab" }
    };
    
    private class Car
    {
        public int Id { get; set; }
        public string Name { get; set; }

    }

    protected override void OnInitialized()
    {
        var dataFromMyFormModel = new List<Car> {

            new Car { Id = 8, Name = "Peugeot" },
            // If you add this back it works
            // new Car { Id = 6, Name = "Volkswagen" }
        };

        SelectedCars = dataFromMyFormModel;
    }
}
```
## 🌍 Your Environment
Windows and Edge Browser

**Comment by dvoituron** [comment]
Thank you for your example of how to reproduce this issue. I appreciate it.

This problem has already been corrected with this PR #2906 and will be included in the next Release.

https://github.com/user-attachments/assets/6d105762-37b7-41e8-ad9f-5ff16cd7fda8

**Comment by lasbrojen** [comment]
> Thank you for your example of how to reproduce this issue. I appreciate it.
> 
> This problem has already been corrected with this PR [#2906](https://github.com/microsoft/fluentui-blazor/pull/2906) and will be included in the next Release.
> 
>  peek_2.mp4

Ahh - okay 🙂👌
